modules/formatter-jsonapi/index.js

```diff
@@ -49,6 +49,13 @@ module.exports = function (input, opts) {
       }
     });
 
+    // Remove foreign keys from model
```

Comment:
This heuristic is inaccurate, we need to use:
https://github.com/endpoints/endpoints/blob/master/modules/formatter-jsonapi/lib/to_one_relations.js to figure this out.